src/components/atoms/ProjectsSliderSection.tsx

```diff
@@ -58,7 +52,7 @@ interface ProjectsSectionSliderProps {
 
 export const ProjectsSectionSlider: React.FC<ProjectsSectionSliderProps> = ({ projects }) => {
     const [current, setCurrent] = useState(0);
-    const [read, setRead] = useState(false);
+    const read = useState(true);
```

Comment:
read będzie zawsze true. Więc jak to już niepotrzebne to wywal wogle tą zmienną stąd i na dole wywal 
`read && <>....` i zostaw komponent po prostu bez `{}`
https://pl.reactjs.org/docs/hooks-reference.html#usestate